websearch/__main__.py

```diff
@@ -0,0 +1,76 @@
+#!/usr/bin/env python
+
+from http.client import error
+from flask import Flask, redirect
+from gevent.pywsgi import WSGIServer
+from .script import WebSearch
+
+import sys
+
+
+APP = Flask(__name__)
+
+@APP.errorhandler(404)
+def page_not_found(e):
+    return """                                                  
+        Can't find what you want. 
+        Please change the query or the extensions                  
+    """
+    
+@APP.route('/v1/image/<string:query>')
```

Comment:
image and page route return many url, so you should make it plurials, /pages, /images

Reply:
it's ok too.